src/colorizer/CanvasOverlay.ts

```diff
@@ -1,38 +1,96 @@
+import { Vector2 } from "three";
 import { numberToSciNotation } from "./utils/math_utils";
 
-export type ScaleBarOptions = {
-  minWidthPx: number;
+export type StyleOptions = {
```

Comment:
don't export? where else is this used?

Reply:
`CanvasWrapper` does use `StyleOptions` for setting up shared properties. It technically doesn't have to, but it gives nicer DX through type hinting.

Reply:
I didn't see an import for it

Reply:
hubris